integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -242,7 +243,7 @@ func createClusterWithNodegroups(clusterName, stackName, ng1, ng2 string, ctl ap
 		})
 		Expect(err).NotTo(HaveOccurred())
 		return *out.Cluster.Status
-	}, time.Minute*20, time.Second*30).Should(Equal("ACTIVE"))
+	}, timeoutDuration, time.Second*30).Should(Equal("ACTIVE"))
```

Comment:
Not introduced in this PR, but we can also use a constant for the second argument.